### ISSUE TITLE
Suggestion: Have z-index controlled by props

### ISSUE THREAD

**Issue by chunghsheng000** [bokuweb/react-rnd#145]
In some case, it will have multiple react-rnd on the screen, different react-rnd has different z-index, and all react-rnd z-index will keep changing depended on which react-rnd the user is currently focusing.

```
        const innerStyle = {
            boxStyle,
            zIndex: (typeof this.props.zIndex) === 'undefined'? this.state.z : this.props.zIndex,
            cursorStyle,
        };
```

In this case, developer could update all react-rnd z-index immediately by passing zIndex to the component. 
If zIndex is not defined, use default state.z.

**Comment by bokuweb** [comment]
@chunghsheng000 tahnks for your report. it is bug , and already fixed at v5.0.9.

https://github.com/bokuweb/react-rnd/blob/master/src/components/index.js#L159-L164

Please use v5.0.9 and `props.z`. 🙏

**Comment by chunghsheng000** [comment]
It works, thank you
